templates/job.njk

```diff
@@ -28,7 +28,7 @@
       <h2>Cómo aplicar</h2>
       {{howToApply | safe }}
       </section>
-      {% if sourceUrl %}
+      {% if sourceUrl !== "http://" %}
```

Comment:
Could you change it to check if it has value and it is different that "http://"? That way it'll be more robust for when we fix the api to return a null value there. Thx!

Reply:
I tried, but i am not very experienced with njk. I could not do a & & comparision. If you don't mind i will have a look tomorrow

Reply:
It's very possible that option is not available in the template, as templates should be very logic-free. A better approach would be to sanitize the data instead. The data for jobs is read and assigned here: https://github.com/VLCTechHub/VLCTechHub-site/blob/master/lib/jobs.js#L37